### ISSUE TITLE
carnival theme: table investigation 

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#1174]
<img width="483" alt="Screen Shot 2019-07-01 at 10 18 52 AM" src="https://user-images.githubusercontent.com/11299125/60447718-a4700200-9be9-11e9-8919-44a30d27ae7b.png">

Investigate if we can separate table title, caption, number elements with new recipe styles. Please do this before we do these: 
https://github.com/openstax/cnx-recipes/issues/1166
https://github.com/openstax/cnx-recipes/issues/1080

In the CNXML these are tagged with `<table><title>TITLE GOES HERE</title><caption>boop boop caption</caption></table>`. 

If it's a huge struggle, we can abandon ship.

**Comment by openstaxalina** [comment]
Link to module with test content: https://easyvm5.cnx.org/contents/x_mdG1WG

**Comment by helenemccarron** [comment]
Linked issue: https://github.com/openstax/cnx-recipes/issues/1241

**Comment by lwtchu** [comment]
I think we're going to keep tables as is for the science theme for a number of reasons.
1. Test load will be too high.
2. Looks like content was specifically crafted for the tables to have top titles and a number on the bottom. We wouldn't be content ready anyway for this styling.
3. We will keep this for the social science theme since it is more prevalent there.
4. We can still apply this to the science theme but we should slowly test content and see if there's any adverse changes. 

Class for this style could be `top-numbertitle`???

tl;dr: we're _tabling_ this for reasons.

**Comment by openstaxalina** [comment]
reopening to connect PR.  We might want this for the social science theme.

**Comment by brittweinstein** [comment]
@lwtchu is this card captured in your backlog of madmen issues?

**Comment by lwtchu** [comment]
@brittweinstein ummm no cause i'm not there yet. let's close this for rn.
